build/libcrypt.ahk

```diff
@@ -1,1221 +1,1381 @@
-LC_Version := "0.0.24.03"
```

Comment:
This should be updated once by simply running `make.ahk` once.

Reply:
I did use the make.ahk, and this was the output I received. Did I need to change the version file or something?

Reply:
Whoops, my bad. You're right, there is a file called "VERSION". I'd say increment it to "0.0.24.04" ?